docker/dsl/build_pipeline.groovy

```diff
@@ -1,7 +1,7 @@
 pipelineJob('build-pipeline-job') { 
 
   // This is the repo that contains the Jenkinsfile you want to use to define your pipeline
-  def repo = 'https://github.com/jschmersal-cscc/special-topics-labs-ci'
+  def repo = 'https://github.com/Felixda1/special-topics-labs-ci.git'
```

Comment:
This is good.  It's using your repo.  However, if you look down a bit you'll see that it's still saying to use `master`.  That's why when I run your pipeline I see:
```
Fetching upstream changes from https://github.com/Felixda1/special-topics-labs-ci.git
 > git fetch --tags --progress https://github.com/Felixda1/special-topics-labs-ci.git +refs/heads/*:refs/remotes/origin/*
 > git rev-parse origin/master^{commit} # timeout=10
Checking out Revision b71084505a94af790d06b8d1d3269adc6a46f587 (origin/master)
```

Note that it's checking out `master` from your repository so it's not picking up the rest of your changes.